### ISSUE TITLE
Floating times not interpreted correctly

### ISSUE THREAD

**Issue by franc6** [franc6/ics_calendar#36]
@franc6 

I'm seeing the same problem
```
- platform: ics_calendar
  calendars:
      - name: "RAFC"
        parser: ics
        url: "https://www.voetbalkrant.com/soccer/calendar/team/team_2_nl.ics"
```

```
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//team_2_nl//NONSGML kigkonsult.se iCalcreator 2.20.2//
X-WR-TIMEZONE:Europe/Brussels
X-WR-CALNAME:Antwerp
X-WR-CALDESC:Bekijk de volledige kalender van voetbalploeg Antwerp uit Onbe
 kend.
...
BEGIN:VEVENT
UID:match_1025179
DTSTAMP:20210911T014015Z
DESCRIPTION:Wedstrijd van Speeldag 1 uit de Europa League  tussen Olympiako
 s Piraeus en Antwerp.\n\nOlympiakos Piraeus wint @ 1.76\nGelijk @ 4.00\nAn
 twerp wint @ 4.35\nWed op https://banners.livepartners.com/click.php?id=34
 9374&p=10&t=betslip&MasterEventID=24267084\n\nOnderlinge duels:\n\n09/12/2
 021 Antwerp - Olympiakos Piraeus: 0-0\n16/09/2021 Olympiakos Piraeus - Ant
 werp: 0-0
DTSTART:20210916T210000
DTEND:20210916T224500
SUMMARY:Olympiakos Piraeus - Antwerp
END:VEVENT
```

<img width="367" alt="Schermafbeelding 2021-09-13 om 15 50 49" src="https://user-images.githubusercontent.com/17234951/133095808-7cf65049-a83b-4cfc-9140-c100a86e4c7e.png">

While the start time must be 21u

I've tried with and without ` parser: ics`

```
homeassistant:
  # Name of the location where Home Assistant is running
  name: 79
  # Location required to calculate the time the sun rises and sets
  latitude: !secret latitude
  longitude: !secret longitude
  # Impacts weather/sunrise data (altitude above sea level)
  elevation: !secret elevation
  # 'metric' for Metric, 'imperial' for Imperial
  unit_system: metric
  # Pick yours from here: http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  time_zone: Europe/Brussels
```

_Originally posted by @gieljnssns in https://github.com/franc6/ics_calendar/issues/6#issuecomment-918214636_

**Comment by gieljnssns** [comment]
Do you have any updates to this issue? 
Or is there something I can change to test?

**Comment by franc6** [comment]
I'm sorry for the delay in a reply.  I'm starting to get some time to work on this project again.  I hope to work on this again before the end of the month.  Both parsers have pre-release versions which include several fixes for other reported problems, and I need to be determine if this issue is my bug or a parser bug.  Plus, the fix for #38 isn't trivial, but very high priority, as it affects everyone.

Thank you for your patience!

**Comment by franc6** [comment]
This should be fixed with the rie parser.  I believe it also works with the ics parser, when 0.8.0 is made available.
